### PR TITLE
[ADD] default_warehouse_from_sale_team: Adding test unit for validation in policy from sale

### PR DESCRIPTION
**Pull request by deivislaya**
Verify that the policy is created with the daily defined in the sales team for that stock

This Pr is related #1062

### CODE REVIEW

**Comment by deivislaya** [comment]
Hi @JulioSerna 

This PR is ready for review

verify policy

File: default_warehouse_from_sale_team/__openerp__.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     'version': '8.0.2.0.0',
     'depends': [
         'sale_stock',
+        'delivery',
```
Comment (JulioSerna) [inline default_warehouse_from_sale_team/__openerp__.py:17]: Why you need this depends?

File: default_warehouse_from_sale_team/tests/test_default_warehouse.py [hunk 2]
```diff
@@ -107,3 +115,63 @@ def test_proper_behavior(self):
         self.assertEqual(sale_brw2.warehouse_id.id, test_wh.id,
                          'Default warehouse is not the warehouse'
                          'set on the sales team realted to de user.')
+
+    def test_warehouse_tean_sale(self):
```
Comment (JulioSerna) [inline default_warehouse_from_sale_team/tests/test_default_warehouse.py]: do you mean test_warehouse_**team**_sale?

**Comment by JulioSerna** [comment]
could you add PR dummy in this pr?

File: default_warehouse_from_sale_team/__openerp__.py (hunk 1 above)
Comment (deivislaya) [reply default_warehouse_from_sale_team/__openerp__.py:17]: The field  `weight_uom_id` is charged for default in this module, without `delivery` depends, a bug occurs `"null value in column "weight_uom_id" violates not-nu
ll constraint`

**Comment by deivislaya** [comment]
Hi @JulioSerna 

This Pr is ready for review

**Comment by deivislaya** [comment]
HI @JulioSerna 

The dummy fail becose the test un other modules related  with sale order get value:

sale_team_user

**Comment by JulioSerna** [comment]
👍

File: default_warehouse_from_sale_team/__openerp__.py (hunk 1 above)
Comment (moylop260) [reply default_warehouse_from_sale_team/__openerp__.py:17]: FYI I added a comment with link to https://github.com/odoo/odoo/pull/13458

**Pull request merged**